### PR TITLE
[RHACS] [rhacs-docs-4.3] ROX-22834: Adding command options for roxctl image and debug commands

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.3

Issue:
https://issues.redhat.com/browse/ROX-22834

Link to docs preview:
- [Checking policy compliance](https://73600--docspreview.netlify.app/openshift-acs/latest/cli/checking-policy-compliance)
- [Debugging issues](https://73600--docspreview.netlify.app/openshift-acs/latest/cli/debugging-issues)

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-4.3`.
- Cherry-picked from https://github.com/openshift/openshift-docs/pull/73457

### CODE REVIEW

**Comment by kcarmichael08** [comment]
LGTM!

File: modules/roxctl-image-command-options-inherited-from-the-parent-command.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+|Force the use of HTTP/1 for all connections. Alternatively, by setting the `ROX_CLIENT_FORCE_HTTP1` environment variable to `true`, you can force the use of HTTP/1. The default value is `false`.
+
+|`--insecure` 
+|Enable insecure connection options. Alternatively, by setting the `ROX_INSECURE_CLIENT` environment variable to `true`, you can enable insecure connection options. The default value is `false`.
+
+|`--insecure-skip-tls-verify` 
+a|Skip the TLS certificate validation. Alternatively, by setting the `ROX_INSECURE_CLIENT_SKIP_TLS_VERIFY` environment variable to `true`, you can skip the TLS certificate validation. The default value is `false`.
+
+|`--no-color` 
+|Disable the color output. Alternatively, by setting the `ROX_NO_COLOR environment variable` to `true`, you can disable the color output. The default value is `false`.
```
Comment (janisz) [inline modules/roxctl-image-command-options-inherited-from-the-parent-command.adoc:35]: I think we also support `NO_COLOR` as the lib we are using supports it

**Review by janisz — APPROVED** [review]

File: modules/roxctl-image-command-options-inherited-from-the-parent-command.adoc (hunk 1 above)
Comment (agantony) [reply modules/roxctl-image-command-options-inherited-from-the-parent-command.adoc:35]: Thanks for the suggestion, but I am not sure if I undertood your comment properly. I have the used the text that is generated in the output.

[agantony@agantony-thinkpadt14sgen2i Downloads]$ ./roxctl -h
Usage:
  ./roxctl [command]

Available Commands:
  central            Commands related to the Central service.
  cluster            Commands related to a cluster.
  collector          Commands related to the Collector service.
  completion         Generate shell completion scripts.
  connectivity-map   (Technology Preview) Analyze connectivity based on network policies and other resources.
  declarative-config Commands that help manage declarative configuration
  deployment         Commands related to deployments.
  generate           (Technology Preview) Commands related to generating different resources.
  helm               Commands related to StackRox Helm Charts.
  help               Help about any command
  image              Commands that you can run on a specific image.
  netpol             (Technology Preview) Commands related to network policies.
  scanner            Commands related to the Scanner service.
  sensor             Commands related to deploying StackRox services in secured clusters.
  version            Display the current roxctl version.

Flags:
      --ca string                  Path to a custom CA certificate to use (PEM format). Alternatively pass the file path using the ROX_CA_CERT_FILE environment variable
      --direct-grpc                Use direct gRPC (advanced; only use if you encounter connection issues). Alternately, enable by setting the ROX_DIRECT_GRPC_CLIENT environment variable to true (default false) (default false)
  -e, --endpoint string            endpoint for service to contact. Alternatively, set the endpoint via the ROX_ENDPOINT environment variable (default "localhost:8443")
      --force-http1                Always use HTTP/1 for all connections (advanced; only use if you encounter connection issues). Alternatively, enable by setting the ROX_CLIENT_FORCE_HTTP1 environment variable to true (default false) (default false)
  -h, --help                       help for ./roxctl
      --insecure                   Enable insecure connection options (DANGEROUS; USE WITH CAUTION). Alternatively, enable insecure connection options by setting the ROX_INSECURE_CLIENT environment variable to true (default false) (default false)
      --insecure-skip-tls-verify   Skip TLS certificate validation. Alternatively, disable TLS certivicate validation by setting the ROX_INSECURE_CLIENT_SKIP_TLS_VERIFY environment variable to true (default false) (default false)
      **--no-color                   Disable color output. Alternately disable the color output by setting the ROX_NO_COLOR environment variable (default false) (default false)**
  -p, --password string            password for basic auth. Alternatively, set the password via the ROX_ADMIN_PASSWORD environment variable
      --plaintext                  Use a plaintext (unencrypted) connection; only works in conjunction with --insecure. Alternatively can be enabled by setting the ROX_PLAINTEXT environment variable to true (default false) (default false)
  -s, --server-name string         TLS ServerName to use for SNI (if empty, derived from endpoint). Alternately, set the server name via the ROX_SERVER_NAME environment variable
      --token-file string          Use the API token in the provided file to authenticate. Alternatively, set the token via the ROX_API_TOKEN environment variable
      --use-current-k8s-context    Use the current kubeconfig context to connect to the central service via port-forwarding. Alternatively, set ROX_USE_KUBECONTEXT environment variable to true (default false) (default false)

**Comment by kcarmichael08** [comment]
Needs a rebase, then can be merged.

**Pull request merged**